ResearchKit/Common/ORKFormStep.m

```diff
@@ -92,31 +92,36 @@ - (void)validateIdentifiersUnique {
 - (instancetype)copyWithZone:(NSZone *)zone {
     ORKFormStep *step = [super copyWithZone:zone];
     step.formItems = ORKArrayCopyObjects(_formItems);
+    step.footnote = self.footnote;
```

Comment:
````
@property (nonatomic, copy, nullable) NSString *footnote;
````

Already does this. What you are suggesting would do the following:
````
step->_footnote = [[self.footnote copy] copy];
````

This is different from the `ORKArrayCopyObjects` macro which copies the objects in the array vs. `step.formItems = self.formItems` which just copies the pointers to the existing objects.